lib/src/components/NxTransferList/TransferListHalf.tsx

```diff
@@ -27,11 +39,36 @@ function TransferListItem<T extends string | number = string>(props: TransferLis
 
   return (
     <NxOverflowTooltip>
-      <label className="nx-transfer-list__item">
-        <NxFontAwesomeIcon icon={checked ? faTimesCircle : faPlusCircle} />
-        <input className="nx-transfer-list__checkbox" type="checkbox" checked={checked} onChange={onChange} />
-        <span>{displayName}</span>
-      </label>
+      <div className={classnames('nx-transfer-list__item',
+          { 'nx-transfer-list__item--with-reordering': !!showReorderingButtons })}>
```

Comment:
Since this `classnames` call is decently complex and multiple lines long, I'd prefer it be pulled out to a variable

Reply:
> Since this `classnames` call is decently complex and multiple lines long, I'd prefer it be pulled out to a variable

Moved it to its own variable here: 774e9702b6a1123bac0705354974edd6087775a5

Reply:
> Moved it to its own variable here: 774e970

I meant move the `classnames` invocation itself.  Like `const classes = classnames('foo', className, { ... });`

That's how we typically do it

Reply:
> > Moved it to its own variable here: [774e970](https://github.com/sonatype/sonatype-react-shared-components/commit/774e9702b6a1123bac0705354974edd6087775a5)
> 
> I meant move the `classnames` invocation itself. Like `const classes = classnames('foo', className, { ... });`
> 
> That's how we typically do it

Updated it here: a233e5ac88b0924c468d44ba1c0abc2a8d5a7083 👍